### ISSUE TITLE
Cell without border when using Boostrap 4

### ISSUE THREAD

**Issue by DavidDuranPerez** [adazzle/react-data-grid#1069]
- Have you already searched for similar issues? Please help us out and double-check first!

- Also, please don't be that person who deletes this template. It's here for a reason.

- Thanks!

---

### Which version of React JS are you using?

✅ Officially supported ✅
- [x ] v15.4.x

⚠️ Not officially supported, expect warnings ⚠️
- [ ] v15.5.x
- [ ] v15.6.x

☣️ Not officially supported, expect warnings and errors ☣️
- [ ] v16.x.x

---

### Which browser are you using?

✅ Officially supported ✅
- [ ] IE 9 / IE 10 / IE 11
- [ ] Edge
- [x ] Chrome

⚠️ Not officially supported, but "should work" ⚠️
- [ ] Firefox
- [ ] Safari

---

### I'm submitting a ...

- [x ] 🐛 Bug Report
- [ ] 💡 Feature Request

> 👋 Need general support? Not sure about how to use React itself, or how to get started with the Grid?
> Please do not submit support request here. Instead see

> https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md

---

When I try to use the React Data Grid with Bootsrap 4, the cells do not have a border when clicking on them. All the other functionalities seem to work correctly.

The script that I put at the "head" section is the following:

https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css

Is this a known issue? 

When I use https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css instead, the border appears. But I need to use Boostrap 4 for other functionalities.

Do you know if there is a workaround for this problem?

```
Please include:
- What the current behavior is
Border of the cells do not appear when clicking

- What the desired behvaior is
That the border of the cell appears when clicking on a cell

- (If Bug) Steps to reproduce the issue
Use Boostrap 4 instead of 3

- (If Feature) The motivation / use case for the feature

We especially love screenshots / videos of problems, and remember
The Best Issue Is A Pull Request™
```

**Comment by JacobCortesAmador** [comment]
I have solved this issue. In Bootstrap 4 there is a file "_reboot.scss" that inside contains this function:

[tabindex="-1"]:focus {
  outline: 0 !important;
}

This function supresses the outlines of the selected cell.

I have created a separeted css file with the following lines:

.react-grid-Cell:focus {
    outline: 2px solid #66afe9 !important;
    outline-offset: -2px !important;
}

These lines overwrite the first function and put the outlines in the selected cell.
